chatgpt-shell.el

```diff
@@ -199,28 +199,22 @@ Can be used compile or run source block at point."
                                   (cons (const primary-action) (function :tag "Action:"))))
   :group 'chatgpt-shell)
 
-(defcustom chatgpt-shell-model-versions
+(defcustom chatgpt-shell-models
   (append chatgpt-shell-openai-models
           chatgpt-shell-anthropic-models
           chatgpt-shell-google-models)
-  "The list of ChatGPT OpenAI models to swap from.
+  "The list of models to swap from.
 
-The list of models supported by /v1/chat/completions endpoint is
-documented at
-https://platform.openai.com/docs/models/model-endpoint-compatibility."
+The list of models supported by chatgpt-shell.el."
   :type '(repeat (alist :key-type symbol :value-type sexp))
   :group 'chatgpt-shell)
 
 (defcustom chatgpt-shell-model-version 0
```

Comment:
ref to the No. 2 in [this comment](https://github.com/xenodium/chatgpt-shell/issues/217#issuecomment-2487452256). I think we can change this custom to, like `chatgpt-shell-model-selection`? But I am pretty happy with the current situation.